Remotion/Core/Core/Context/SafeContext.Task.cs

```diff
@@ -27,6 +28,7 @@ public static partial class SafeContext
     /// Provides methods for running <see cref="SystemTask"/>s which are <see cref="SafeContext"/> aware.
     /// <see cref="SafeContext"/> values do not flow into the executed tasks.
     /// </summary>
+    [SuppressMessage("Usage", "RMCORE0001:Use SafeContext instead of typical API")]
```

Comment:
@a-ctor 
cc @Frazerus 

I tend to lean towards SuppressMessageAttribute, mainly because I like attributes and it does seem the "correct" choice. I would like to do a quick discussion of the attribute and the pros and cons after the Standup and then document the findings here to make sure it's not just a gut desicion.